mailbox/lucene/src/main/java/org/apache/james/mailbox/lucene/search/LuceneMessageSearchIndex.java

```diff
@@ -1241,36 +1240,6 @@ public void add(MailboxSession session, Mailbox mailbox, MailboxMessage membersh
         writer.addDocument(flagsDoc);
     }
 
-    @Override
-    public void update(MailboxSession session, Mailbox mailbox, List<UpdatedFlags> updatedFlagsList) throws IOException {
```

Comment:
Why do you change Lucene related code in an ElasticSearch pull request?

This should be a warning sign that you are doing something wrong.